lib/rules/no-multiple-empty-lines.js

```diff
@@ -34,6 +46,12 @@ module.exports = function(context) {
                 trimmedLines = lines.map(function(str) {
                     return str.trim();
                 });
+
+            // add the notEmpty lines in there with a placeholder
+            notEmpty.forEach(function(x, i) {
```

Comment:
I wonder if it would be easier to take trimmedLines and get nodes for corresponding location, and check if node is TemplateLiteral instead of doing those two loops. I think in the end it doesn't matter a whole lot, but it would probably take big o down by a few n's.